Formula/avr-gcc.rb

```diff
@@ -80,6 +94,7 @@ def install
 
       build = `./config.guess`.chomp
 
+      system "./bootstrap" if localBuild.with? "ATMega168pbSupport"
```

Comment:
can't we just do?

```ruby
system "./bootstrap" if build.with? "ATMega168pbSupport"
```

Reply:
Same here, this snipped is inside a resource entity and if i write the expression like you did, homebrew will throw me an exception.

Reply:
If using 
system "./bootstrap" if build.with? "ATMega168pbSupport"
i got
NoMethodError: undefined method `with?' for "i686-apple-darwin18.5.0":String